src/context.rs

```diff
@@ -508,11 +508,11 @@ pub fn has_chroma(
 pub fn get_tx_set(
   tx_size: TxSize, is_inter: bool, use_reduced_set: bool
 ) -> TxSet {
-  let tx_size_sqr_up = tx_size.sqr_up();
   let tx_size_sqr = tx_size.sqr();
-  if tx_size_sqr > TxSize::TX_32X32 {
+
+  if tx_size.width() >= 32 || tx_size.height() >= 32 {
```

Comment:
Ooops, I should use && instead of || here, so that any tx_size having 64 as one dim use dct_dct only.

Reply:
I think || is fine. Shouldn't it be > instead of >=?

Reply:
Hmmm, I thought wrong again.  It should be:  w > 32 || h > 32. or w >= 64 || h >= 64

Reply:
Yes! I was writing that! :) thx-